src/Component/Header/Header.jsx

```diff
@@ -0,0 +1,18 @@
+
+import github from "../../Assest/github.jpeg"
+
+export const Header = () => {
+    return (
+      <div>
+        <nav className="mt-4 w-3/4 mx-auto">
+          <div className='flex justify-between items-center'>
+            <img src={github} alt='github-icon' className='w-24 h-16' />
+            <p className="hover:text-gray-300 text-sm">Log in/Register</p>
```

Comment:
I am not sure we need this Login/register, the task does not require us to login or register

Reply:
Sure, I understand that adding the login/register wasn't necessary and since the instructions said there's no specific right or wrong way to do the exercise. I just wanted to add something there to avoid the page looking too empty, but I'll go ahead and remove it

Reply:
Sweeeeeet.